code/modules/hydroponics/seed.dm

```diff
@@ -47,6 +47,7 @@
 	set_trait(TRAIT_PLANT_ICON,           0)            // Icon to use for the plant growing in the tray.
 	set_trait(TRAIT_PRODUCT_COLOUR,       0)            // Colour to apply to product icon.
 	set_trait(TRAIT_BIOLUM_COLOUR,        0)            // The colour of the plant's radiance.
+	set_trait(TRAIT_RARITY,				  0)			// How rare the plant is. Used for giving points to cargo when shipping off to Centcom.
```

Comment:
pretty sure all the other ones were actually just spaced out to that level, given that github is displaying this wrong